### DISCUSSION TITLE
Are we planning to have the status checks driven from Github?

### DISCUSSION THREAD

**Discussion by fokion** [ovh/venom#702] [Q&A]
Is there a plan to have the status checks and testing in general to be handled in github actions so that others can have visibility what is failing? I know that we are currently using the OVH's solution for CI/CD but in that case should the page with those execution be public?

**Comment by yesnault** [comment]
Hi @fokion , for now, we have no plan about that, but you can use gitpod to run integration tests easily (make run-test, with all dependencies): https://www.gitpod.io/ https://github.com/ovh/venom/blob/master/.gitpod.yml
When it's failed on CDS, we watch the output and report errors if necessary.
